tests/common.go

```diff
@@ -423,6 +430,12 @@ func InitInstance() {
 	err = Inst().M.Init(Inst().JobName, Inst().JobType)
 	log.FailOnError(err, "Error occured while monitor Initialization")
 
+	if Inst().Pds != nil {
+		log.InfoD("Initializing PDS dataservice methods")
```

Comment:
make them infof

Reply:
make it as "PDS dataservice initialised"

Reply:
done